### ISSUE TITLE
Error assembling

### ISSUE THREAD

**Issue by YuanwenGuo** [PapenfussLab/gridss#242]
Hi,
I am running GRIDSS with multiple input samples. All of them work well except one sample shows error message:

ERROR   2019-09-04 10:47:26     NonReferenceContigAssembler     Sanity check failure: found path with no support. Attempting to recover by direct node removal scaffold_81
WARNING 2019-09-04 10:47:26     AssemblyAttributes      Sanity check failure: asm18-9041 has mapq below minimum
ERROR   2019-09-04 10:47:26     NonReferenceContigAssembler     Sanity check failure: found path with no support. Attempting to recover by direct node removal scaffold_81
ERROR   2019-09-04 10:47:26     PositionalAssembler     Error assembling scaffold_81. This should not happen. Please raise an issue at https://github.com/PapenfussLab/gridss/issues. At
tempting recovery by resuming assembly at scaffold_81:18618
au.edu.wehi.idsv.SanityCheckFailureException: Subnode [8892-8912][8892-9270]  845w GTTAAGGTAAATGTAGGAGGGATAT ATTTAGGCATAGTAACGTGGCGAGGTAGTTAAACGATACGAGACAAGATGTGACAGAGGGGGTG (65)  reac
hable from score=130 pathlength=10 [8882-8902][8882-8902]  130w GTATATTTTTGTTAAGGTAAATGTA GGAGGGATA (10)  not memoized. [519 nodes memoized, 12 in frontier, maxVisitedEndPosition=12085
, nextPosition=12099]
        at au.edu.wehi.idsv.debruijn.positional.MemoizedContigCaller.visit(MemoizedContigCaller.java:205)
        at au.edu.wehi.idsv.debruijn.positional.MemoizedContigCaller.advanceFrontier(MemoizedContigCaller.java:183)
        at au.edu.wehi.idsv.debruijn.positional.MemoizedContigCaller.callBestContigStartingBefore(MemoizedContigCaller.java:335)
        at au.edu.wehi.idsv.debruijn.positional.NonReferenceContigAssembler.flushCallsOutsideRetainWindow(NonReferenceContigAssembler.java:197)
        at au.edu.wehi.idsv.debruijn.positional.NonReferenceContigAssembler.ensureCalledContig(NonReferenceContigAssembler.java:232)
        at au.edu.wehi.idsv.debruijn.positional.NonReferenceContigAssembler.hasNext(NonReferenceContigAssembler.java:169)
        at au.edu.wehi.idsv.debruijn.positional.PositionalAssembler.flushIfRequired(PositionalAssembler.java:69)
        at au.edu.wehi.idsv.debruijn.positional.PositionalAssembler.ensureAssembler(PositionalAssembler.java:109)
        at au.edu.wehi.idsv.debruijn.positional.PositionalAssembler.ensureAssembler(PositionalAssembler.java:85)
        at au.edu.wehi.idsv.debruijn.positional.PositionalAssembler.hasNext(PositionalAssembler.java:55)
        at au.edu.wehi.idsv.AssemblyEvidenceSource.assembleChunk(AssemblyEvidenceSource.java:220)
        at au.edu.wehi.idsv.AssemblyEvidenceSource.assembleChunk(AssemblyEvidenceSource.java:178)
        at au.edu.wehi.idsv.AssemblyEvidenceSource.lambda$assembleBreakends$1(AssemblyEvidenceSource.java:87)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)

I am using version gridss-2.5.2. I will be much appreciate if anyone could help me with this issue.

Thank you!

Best,
Yuanwen

**Comment by d-cameron** [comment]
Was the error fatal? If you have output, then this error is limited to missing contigs around scaffold_81:18618

**Comment by d-cameron** [comment]
Duplicate of #237

**Comment by YuanwenGuo** [comment]
Thank you for prompt response!
Unfortunately I didn't get output vcf files, and seems like GRIDSS stopped at AssembleBreakends step. Following are the last few log lines I have when running GRIDSS:
Max file handles: 131072
Wed Sep  4 10:46:49 CDT 2019    Running GRIDSS. The full log is in ./gridss.full.20190904_104649.elf79.33345.log
Wed Sep  4 10:46:49 CDT 2019    Start pre-processing    /bulk/feihe/data/Pgt_isolates/bam_YuanWen/01MN84-A-1-2_S51.RG.DupMarked.bam
Wed Sep  4 10:46:49 CDT 2019    Skipping pre-processing as ./01MN84-A-1-2_S51.RG.DupMarked.bam.gridss.working/01MN84-A-1-2_S51.RG.DupMarked.bam.sv.bam already exists. /bulk/feihe/data/Pgt_isolates/bam_YuanWen/01MN84-A-1-2_S51.RG.DupMarked.bam
Wed Sep  4 10:46:49 CDT 2019    Start assembly  /bulk/yuanwen/RKQQC/20190830_GRIDSS/files/gridss.full.01MN84-A-1-2_S51.bam
Wed Sep  4 10:46:49 CDT 2019    AssembleBreakends       /bulk/yuanwen/RKQQC/20190830_GRIDSS/files/gridss.full.01MN84-A-1-2_S51.bam

Thank you so much for the help!

Best,
Yuanwen

**Comment by SunWinner01** [comment]
Hello,
I have received the result file, but when I checked the nohup.out file, I also found a similar error. I would like to ask, what should I do next? Should I ignore this error？

**_ERROR   2024-09-20 00:44:41     SingleReadEvidence      Data sanity check failure: split read alignment of A00920:567:HTVJVDSXY:1:1418:18530:30264 from /gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/bam_data/KF10-3.sort.rmdup.bam have different read lengths.  This is typically caused by GATK indel realignment stripping hard clipping from read alignments.  Ignoring read
INFO    2024-09-20 00:44:44     SequentialCoverageAnnotator     Processed    10,000,000 records.  Elapsed time: 00:01:06s.  Time for last 10,000,000:   66s.  Last read position: NC_047562.1:9,394,538
INFO    2024-09-20 00:44:56     SequentialEvidenceAllocator     Processed     4,000,000 records.  Elapsed time: 00:01:18s.  Time for last 1,000,000:   17s.  Last read position: NC_047562.1:49,350,900.  Last read name: A00920:567:HTVJVDSXY:1:2646:9507:30013
INFO    2024-09-20 00:45:14     SequentialEvidenceAllocator     Processed     5,000,000 records.  Elapsed time: 00:01:36s.  Time for last 1,000,000:   18s.  Last read position: NC_047563.1:64,472,284.  Last read name: A00920:567:HTVJVDSXY:1:1621:29749:28150
ERROR   2024-09-20 00:45:17     SingleReadEvidence      Data sanity check failure: split read alignment of A00920:567:HTVJVDSXY:1:1613:16839:9674 from /gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/bam_data/KF10-3.sort.rmdup.bam have different read lengths.  This is typically caused by GATK indel realignment stripping hard clipping from read alignments.  Ignoring read
ERROR   2024-09-20 00:45:20     SingleReadEvidence      Data sanity check failure: split read alignment of A00920:567:HTVJVDSXY:1:1423:30355:32049 from /gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/bam_data/KF10-3.sort.rmdup.bam have different read lengths.  This is typically caused by GATK indel realignment stripping hard clipping from read alignments.  Ignoring read
ERROR   2024-09-20 00:45:26     SingleReadEvidence      Data sanity check failure: split read alignment of A00920:567:HTVJVDSXY:1:2532:3296:13902 from /gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/bam_data/KF10-3.sort.rmdup.bam have different read lengths.  This is typically caused by GATK indel realignment stripping hard clipping from read alignments.  Ignoring read
ERROR   2024-09-20 00:45:26     SingleReadEvidence      Data sanity check failure: split read alignment of A00920:567:HTVJVDSXY:1:2532:4896:14168 from /gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/bam_data/KF10-3.sort.rmdup.bam have different read lengths.  This is typically caused by GATK indel realignment stripping hard clipping from read alignments.  Ignoring read
INFO    2024-09-20 00:45:26     SingleReadEvidence      Suppressing messages regarding SingleReadEvidence.createEvidence() failure
ERROR   2024-09-20 00:45:26     SingleReadEvidence      Data sanity check failure: split read alignment of A00920:567:HTVJVDSXY:1:1448:4625:17550 from /gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/bam_data/KF10-3.sort.rmdup.bam have different read lengths.  This is typically caused by GATK indel realignment stripping hard clipping from read alignments.  Ignoring read
INFO    2024-09-20 00:45:31     SequentialEvidenceAllocator     Processed     6,000,000 records.  Elapsed time: 00:01:53s.  Time for last 1,000,000:   16s.  Last read position: NC_047564.1:44,339,888.  Last read name: A00920:567:HTVJVDSXY:1:2151:31078:27978
INFO    2024-09-20 00:45:42     SequentialCoverageAnnotator     Processed    20,000,000 records.  Elapsed time: 00:02:04s.  Time for last 10,000,000:   58s.  Last read position: NC_047565.1:21,207,299
INFO    2024-09-20 00:45:48     SequentialEvidenceAllocator     Processed     7,000,000 records.  Elapsed time: 00:02:10s.  Time for last 1,000,000:   16s.  Last read position: NC_047565.1:38,721,284.  Last read name: asm41-49404
INFO    2024-09-20 00:46:06     SequentialEvidenceAllocator     Processed     8,000,000 records.  Elapsed time: 00:02:28s.  Time for last 1,000,000:   18s.  Last read position: NC_047566.1:32,544,490.  Last read name: A00920:570:HVWHFDSXY:2:1368:30572:11256
INFO    2024-09-20 00:46:24     SequentialEvidenceAllocator     Processed     9,000,000 records.  Elapsed time: 00:02:46s.  Time for last 1,000,000:   18s.  Last read position: NC_047567.1:32,950,584.  Last read name: A00920:570:HVWHFDSXY:2:2273:8196:28213
INFO    2024-09-20 00:46:41     SequentialEvidenceAllocator     Processed    10,000,000 records.  Elapsed time: 00:03:03s.  Time for last 1,000,000:   16s.  Last read position: NC_047568.1:49,699,209.  Last read name: A00920:567:HTVJVDSXY:1:1228:10013:27477
INFO    2024-09-20 00:46:42     SequentialCoverageAnnotator     Processed    30,000,000 records.  Elapsed time: 00:03:04s.  Time for last 10,000,000:   59s.  Last read position: NC_047568.1:52,698,152
4gridss.sh: line 4: 2956503 Killed                  java -ea -Xmx100g -Dsamjdk.create_index=true -Dsamjdk.use_async_io_read_samtools=true -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=true -Dgridss.gridss.output_to_temp_file=true -cp /gpfshddpool/home/sunweiying/software/swgridss/gridss-2.13.2-gridss-jar-with-dependencies.jar gridss.CallVariants WORKER_THREADS=30 TMP_DIR=/gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/GRIDSS-out/4-2new-kf-working WORKING_DIR=/gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/GRIDSS-out/4-2new-kf-working REFERENCE_SEQUENCE=/gpfshddpool/home/sunweiying/zhangzh/ws1/01.ref/GCF_902806645.1_cgigas_uk_roslin_v1_genomic.fna INPUT=/gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/bam_data/$i.sort.rmdup.bam OUTPUT=/gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/GRIDSS-out/$i.vcf ASSEMBLY=/gpfshddpool/home/sunweiying/node3_sunwy/10x_sv/GRIDSS-out/4-2new-kf-working/4-2new-kf-assembly/$i.gridss.assembly.bam_**



The above is a partial content of my nohup.out. Sincerely hope to receive your reply, thank you. Wishing you all the best
